js/app.js

```diff
@@ -0,0 +1,178 @@
+//Easy Going
+
+// for (i = 1; i <= 20; i++) {
+// 	console.log(i)
+// }
+
+
+//Get Even
+
+// for (i = 0; i <= 200; i += 2) {
+// 	console.log(i)
+// }
+
+
+//Excited Kitten
+
+// const catPhrases = ['...human...why you taking pictures of me?..', '..the catnip made me do it...', '..why does the red dot always get away...'];
+
+// for (i = 0; i < 20; i++) {
+// 	console.log('Love me, pet me! HSSSSSS!')
+// 	if (i % 2 === 0) {
+// 		const randomPhrase = catPhrases[Math.floor(catPhrases.length * Math.random())];
+// 		console.log(randomPhrase);
+// 	}
+// }
+
+//right now this just returns the SAME randomized phrase each time, for all the evens... 
+
+
+//Fizz Buzz
+
+
+// for (i = 1; i < 100; i++) {
+// 		if (i % 3 === 0) {
+// 			console.log('Fizz')
+// 		} if (i % 5 === 0) {
+// 			console.log('Buzz')
+// 		} if (i % 3 === 0 || i % 5 === 0) {
+// 			console.log('Fizzbuzz')
+// 		}
+// }
+
+// ^^ classic original vv displays number alongside Fizz/Buzz/Fizzbuzz - built so I could double check the numbers. 
+
+// for (i = 1; i < 100; i++) {
+// 		if (i % 3 === 0) {
+// 			console.log('Fizz ' + i)
+// 		} if (i % 5 === 0) {
+// 			console.log('Buzz ' + i)
+// 		} if (i % 3 === 0 || i % 5 === 0) {
+// 			console.log('Fizzbuzz ' + i)
+// 		}
+// }
```

Comment:
Are you running your code after you write it?  A quick glance at the output would have told you it isn't working right. Be sure to test your code!!!